src/watchmaker/cli.py

```diff
@@ -15,15 +15,16 @@ def main():
                         ))
     parser.add_argument('--config', dest='config', default='config.yaml',
                         help='Path or URL to the config.yaml file.')
-    parser.add_argument('--logger', dest='logger', action='store_true',
-                        help='Use stream logger for debugging.')
-    parser.add_argument('--log-path', dest='log_path', default=None,
-                        help='Path to the logfile for stream logging.')
     parser.add_argument('--saltstates', dest='saltstates', default=None,
                         help=(
                             'Define the saltstates to use.  Must be None, '
                             'Highstate, or comma-seperated-string'
                         ))
+    group = parser.add_mutually_exclusive_group(required=False)
```

Comment:
Nice, was wondering how we were going to do that, turns out it's just part of argparse. Looking down the road, maybe name this `log_group`, so if we have another set of mutually exclusive params the naming continues to make sense?